css/src/components/input.css

```diff
@@ -115,13 +119,17 @@
 
 .Input-icon--left .Icon {
   margin-right: var(--spacing);
-  color: var(--inverse-lighter);
+  color: var(--inverse);
 }
 
 .Input-icon--disabled .Icon {
   color: var(--inverse-lighter);
 }
 
+.Input-icon--error .Icon {
+  color: var(--alert);
+}
+
```

Comment:
Please remove the nesting `.Icon` it is redundant

Reply:
removing  `.Icon` is preventing icon to receive the desired color.
```